store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/Logger.kt

```diff
@@ -0,0 +1,24 @@
+package org.mobilenativefoundation.store.store5
+
+/**
+ * A simple logging interface for logging error and debug messages.
+ */
+interface Logger {
+    /**
+     * Logs an error message, optionally with a throwable.
+     *
+     * @param message The error message to log.
+     * @param throwable An optional [Throwable] associated with the error.
+     */
+    fun error(
+        message: String,
+        throwable: Throwable? = null,
+    )
+
+    /**
+     * Logs a debug message.
+     *
+     * @param message The debug message to log.
+     */
+    fun debug(message: String)
+}
```

Comment:
no info?

Reply:
We're not currently using info and this component isn't currently being exposed to the consumer. There's an open request to improve logging #638. My plan is to replace this then with logging hooks delegated to the user